tests/sentry/utils/sdk_crashes/test_sdk_crash_detection.py

```diff
@@ -111,615 +105,6 @@ def test_sdks_detected(mock_sdk_crash_reporter, store_event, sdk_name, detected)
         assert mock_sdk_crash_reporter.report.call_count == 0
 
 
-@patch("sentry.utils.sdk_crashes.sdk_crash_detection.sdk_crash_detection.sdk_crash_reporter")
```

Comment:
There is no change in the functionality of the tests. I moved them to another file.